js/src/main/scala/org/widok/bindings/Bootstrap/package.scala

```diff
@@ -141,6 +148,8 @@ package object Bootstrap {
     def element = DOM.createElement("button", contents)
   }
 
+  def SubmitButton(contents: View*) = Button(contents: _*).attribute("type", "submit")
```

Comment:
You are right. But we should probably have Boostrap buttons too, with classes "btn btn-default"?